lms/templates/discussion/_js_head_dependencies.html

```diff
@@ -1,8 +1,9 @@
 <%namespace name='static' file='../static_content.html'/>
 
+<link href="${static.url('css/vendor/jquery.autocomplete.css')}" rel="stylesheet" type="text/css">
```

Comment:
If I'm reading this correctly, all of this file is included in the block ``headextra_css``. That means that there is a mix of JavaScript and CSS going into the CSS block, which is confusing (although obviously it will work). I would think that JS dependencies should be rendered into the ``headextra`` section. In fact, we want to move all JS out of the head as part of this work:

https://openedx.atlassian.net/browse/TNL-2949